### ISSUE TITLE
Exception in erdbeermet.simulation.load

### ISSUE THREAD

**Issue by MalteT** [david-schaller/Erdbeermet#1]
Loading the following file:
```
(0, 0: 1) 1.0; [0.24385458723201128,0.07690366106144374]
(0, 1: 2) 0.45117649394645853; [0.115646069331347,0.07190626246660096,0.45439976814034444]
(0, 2: 3) 7.311325058234619e-05; [0.9762744496297727,0.12312561202542208,0.08077018465583378,0.005984710533442837]
(3, 2: 4) 0.2583920074283106; [0.09446663498367167,0.09453805400419821,0.4352091649154993,0.340788636503409,0.043874091958025455]
(4, 2: 5) 0.7555814973642914; [0.001973313862727302,0.07533437041504709,0.08030210466015564,0.041481102771070186,0.2762702437424781,0.2158712201889108]
```

results in the following exception:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/nix/store/g65qhgcskdh58w5348mbivkdh50vylf1-python3.9-erdbeermet-local/lib/python3.9/site-packages/erdbeermet/simulation.py", line 325, in load
    return scenario_from_history(FileIO.parse_history(filename),
  File "/nix/store/g65qhgcskdh58w5348mbivkdh50vylf1-python3.9-erdbeermet-local/lib/python3.9/site-packages/erdbeermet/simulation.py", line 307, in scenario_from_history
    return Scenario(history[:N-1])
  File "/nix/store/g65qhgcskdh58w5348mbivkdh50vylf1-python3.9-erdbeermet-local/lib/python3.9/site-packages/erdbeermet/simulation.py", line 41, in __init__
    self._build_matrix()
  File "/nix/store/g65qhgcskdh58w5348mbivkdh50vylf1-python3.9-erdbeermet-local/lib/python3.9/site-packages/erdbeermet/simulation.py", line 150, in _build_matrix
    if self._circ_order[x] == y:
KeyError: 3
```

@Darkdragon853

**Comment by david-schaller** [comment]
Which version did you use? The bug fix in v0.0.4 should already solve this. I can load the file correctly.
The problem was a regex that could not handle scientific notation of float numbers for the value of alpha (third line in your example: 7.311325058234619e-05).

**Comment by MalteT** [comment]
Yes... my bad, I had an error in my setup, thanks for the quick response! Works perfectly on v0.0.4
